shared/lib/constants.py

```diff
@@ -151,6 +150,10 @@
     'across',
 }
 
+# For a stop words, if it is preceeded or trailed by certain words, don't trim
+# it.
+STOP_WORDS_EXCEPTION = {'how': ['', 'many'], 'of': ['number', '']}
```

Comment:
This feels hard to use.  Is it worth just listing phrases to keep?  "how many", "number of"